Split/Common/Extensions/Date+Utils.swift

```diff
@@ -55,4 +55,11 @@ extension Date {
     static func interval(millis: Int64) -> Int64 {
         return Date.nowMillis() - millis
     }
+
+    static func nowLabel() -> String {
+        let formatter = DateFormatter()
```

Comment:
The `formatter` variable could be an static instance, so that it is not re-created on each method call

Reply:
Good catch! Thank you Doc!!